src/modules/getData.js

```diff
@@ -0,0 +1,22 @@
+const gameUser = document.getElementById('game-user');
+const gameScore = document.getElementById('game-score');
+const gameScoreBtn = document.getElementById('score-btn');
+
+const myGameUSerData = async () => {
```

Comment:
[OPTIONAL]

I will encourage you to clear the form as soon as the user click submit so as to prevent the user from clicking multiple times which will result to duplicate. So in this function, default the values back to empty.

e.g.
   
  ```
 gameUser.value = ' '
   gameScore.value = ''
```